src/ResourceManager/HDInsight/Commands.HDInsight.Test/ScenarioTests/DataLakeStoreScenarioTests.cs

```diff
@@ -10,11 +10,18 @@ public DataLakeStoreScenarioTests(Xunit.Abstractions.ITestOutputHelper output)
             ServiceManagemenet.Common.Models.XunitTracingInterceptor.AddToContext(new ServiceManagemenet.Common.Models.XunitTracingInterceptor(output));
         }
 
-        [Fact]
+        [Fact (Skip = "Test requires fixing GraphAPI by upgrading to latest test framework")]
```

Comment:
@markcowl , @shahabhijeet .

I had merged from Azure:dev. 
The test framework has been upgraded to Microsoft.Azure.Test.Framework.1.0.6052.28118-prerelease and Graph SDK has been updated to Microsoft.Azure.Graph.RBAC.3.2.0-preview.

However, I am having trouble with getting these upgrades to work with the code.
When I remove the "skip", the NewAzureRmHDInsightCluster cmdlet calls GetApplicationId, which in turn eventually calls MockClientFactory.CreateCustomArmClient<GraphRbacManagementClient>. 
This function tries to find a management client that it can use (TClient client = ManagementClients.FirstOrDefault(o => o is TClient) as TClient;), fails and then tries to create a real-client by reflecting and calling the constructor.
The problem is that it is not able to create a GraphRbacManagementClient because GraphRbacManagementClient does not have the expected constructor.


I then tried to setup the management client such that MockClientFactory.CreateCustomArmClient<GraphRbacManagementClient> will find a client object that it can use directly (and avoid trying to create a real-client via reflection).


I added the code below to HDInsightScenarioTestsBase.cs and expected this to work.
However TestBase.GetGraphServiceClient<GraphRbacManagementClient> is throwing a null reference exception...I am not sure why. I am not able to find the code for the TestBase class.
protected void SetupManagementClients()
        {
            var hdinsightManagementClient = GetHdInsightManagementClient();
            var graphRbacManagementClient = TestBase.GetGraphServiceClient<GraphRbacManagementClient>(this.csmTestFactory);

            helper.SetupSomeOfManagementClients(hdinsightManagementClient, graphRbacManagementClient);
        }


Can you help me out with getting this to work ?

Thanks
Siva